fix: change onboarding

krushnarout opened the pull request:
fixes #2063 
/claim #2063 

https://github.com/user-attachments/assets/cb5c5701-ac48-4f09-8d80-26eac59ed811

beastoin commented:
1/ tell me more about your fixes since Connect to omi -> entering the no device(twitter) flow is really weird, make sure you understand about what should be changed first. if not, lets clarify it. https://github.com/BasedHardware/omi/issues/2063

@krushnarout

krushnarout commented:
> 1/ tell me more about your fixes since Connect to omi -> entering the no device(twitter) flow is really weird, make sure you understand about what should be changed first. if not, lets clarify it. #2063
> 
> @krushnarout

@beastoin Sorry, I missed the Twitter flow earlier. I've fixed it now.

<img width="1470" alt="Screenshot 2025-03-22 at 9 47 34 AM" src="https://github.com/user-attachments/assets/69af1f10-091e-4936-8ec7-3289d50018ea" />

krushnarout commented:
> clarify the flow

@beastoin Done! Please check.

Connect omi ->

https://github.com/user-attachments/assets/5a356767-4682-4439-a8a0-ca9ee2bdc547

I don't have omi ->

https://github.com/user-attachments/assets/71f36fce-ff99-4776-8fb9-5d3de2ed0410

krushnarout commented:
@beastoin Please review! I've made changes according to the given flow, and it’s working fine!

krushnarout commented:
> 1/ ok 2/ why did we need these changes sir ? https://github.com/BasedHardware/omi/pull/2064/files#diff-be78b37404fce4833b2724c446c5e181e04ca6c2c576dfd4f88fa62085642a98L72-L245
> 

@beastoin I made changes according to the design

<img width="656" alt="Screenshot 2025-03-24 at 9 49 41 AM" src="https://github.com/user-attachments/assets/cfc39e6a-21d6-4aa5-b412-c7e34ae6f512" />

beastoin commented:
2/ ahhh please clarify it with Nik if he needs that changes or not. since for me this task should be the flow.

@krushnarout 

and @mdmohsin7 feel free to help with merging ;)

krushnarout commented:
@kodjima33 I updated the flow as per the issue and removed the robot icon based on the design. Just wanted to confirm if the robot icon removal is required.

beastoin commented:
ahh, lfg / lgtm @krushnarout . btw if we have any updates then i will let you know :)

krushnarout commented:
> ahh, lfg / lgtm @krushnarout . btw if we have any updates then i will let you know :)

Sounds good! Let me know if anything comes up

krushnarout commented:
> ahh, lfg / lgtm @krushnarout . btw if we have any updates then i will let you know :)

@beastoin Sounds good! Let me know if anything comes up

krushnarout commented:
Hi @mdmohsin7 , the bounty has been completed and accepted. Just pinging as requested to receive the payment. Thanks!
